src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/OffenderSearch.kt

```diff
@@ -4,9 +4,9 @@ import java.time.LocalDate
 
 data class OffenderSearchPrisoner(
   val prisonerNumber: String,
-  val bookingId: String,
+  val bookingId: Long,
```

Comment:
offender search returns this as a string, will jakson (or whatever) correctly convert it

Reply:
yes -it does